docs/content/ingestion/native_tasks.md

```diff
@@ -28,6 +28,9 @@ Druid currently has two types of native batch indexing tasks, `index_parallel` w
 in parallel on multiple MiddleManager nodes, and `index` which will run a single indexing task locally on a single
 MiddleManager.
 
+To run either kind of native batch indexing task, write an ingestion spec as specified below. Then POST it to the
+`/druid/indexer/v1/task` endpoint on your Overlord machine, or use the `post-index-task` script included with Druid.
```

Comment:
Please add a link to http://druid.io/docs/latest/operations/api-reference.html#tasks for details of the API.